app.rb

```diff
@@ -0,0 +1,112 @@
+require './book'
+require './person'
+require './rental'
+require './teacher'
+require './student'
+
+class App
+  def on_start
+    puts '--------------------------------------------------'
+    puts 'Select an option: '
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals'
+    puts '7 - Exit'
+    puts '--------------------------------------------------'
+  end
```

Comment:
Your app is currently not storing the data anywhere so it closes when you create a person or book. As a result it becomes difficult to go ahead and create  rental or even list available books/people. 

- [ ] Kindly consider initializing the books, people, and rentals and storing them in an array to fix this issue. 
- [ ] You should also check and make sure to the app can list all books and able to undergo all functions successfully. :pray: 

![Screenshot from 2023-01-06 23-14-12](https://user-images.githubusercontent.com/65233860/211115565-84d378f8-4b65-4934-880b-94b1dd4e4d51.png)

![Screenshot from 2023-01-06 23-14-19](https://user-images.githubusercontent.com/65233860/211115569-2c525b39-e923-420d-ade2-ccfd95d61a09.png)